server/chat-commands/room-sections.ts

```diff
@@ -0,0 +1,94 @@
+import {Utils} from '../../lib';
+
+export type RoomSection = 'none' | 'nonpublic' | 'officialrooms' | 'officialtiers' | 'communityprojects' |
```

Comment:
Is there a reason this is a new file? It feels better suited to move the types / sections into rooms.ts, and the commands into room-settings (seeing as this _is_ a setting)

Reply:
(to clarify, almost everything in this file is exported and interacts directly with the client, so i secluded it for cleanliness)

Reply:
Unresolving this since it appears I am not the only one with this question: